src/Stormpath/Resource/VerificationEmailRequest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Stormpath\Resource;
+
+/*
+ * Copyright 2013 Stormpath, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+class VerificationEmailRequest
```

Comment:
Now that this class does not accept the account store option, I am not sure we even need it anymore.  Why would we need a class that just has the one property and not just pass in the email directly to the sendVerificationEmail.  What do you think @ecrisostomo